presto-main/src/main/java/com/facebook/presto/execution/TaskStatus.java

```diff
@@ -86,7 +88,8 @@ public TaskStatus(
             @JsonProperty("fullGcCount") long fullGcCount,
             @JsonProperty("fullGcTimeInMillis") long fullGcTimeInMillis)
     {
-        this.taskInstanceId = requireNonNull(taskInstanceId, "taskInstanceId is null");
+        this.taskInstanceIdLsb = taskInstanceIdLsb;
```

Comment:
let's not abreviate, call it `LeastSignificantBits`

Reply:
done